apps/test/unit/code-studio/components/progress/LessonProgressTest.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {assert} from 'chai';
+import {UnconnectedLessonProgress as LessonProgress} from '@cdo/apps/code-studio/components/progress/LessonProgress';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+import {shallow} from 'enzyme';
+
+describe('LessonProgress', () => {
```

Comment:
Thank you for breaking these renames into individual commits! It's odd that github doesn't recognize this file as having been renamed.